WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoPickerAdapter.java

```diff
@@ -125,23 +96,7 @@ public void onBindViewHolder(@NonNull ThumbnailViewHolder holder, int position,
 
     @Override
     public void onBindViewHolder(@NonNull ThumbnailViewHolder holder, int position) {
-        PhotoPickerUiItem item = getItemAtPosition(position);
-        if (item == null) {
-            return;
-        }
-        holder.bind(item, false, false);
-    }
-
-    private PhotoPickerUiItem getItemAtPosition(int position) {
-        if (!isValidPosition(position)) {
-            AppLog.w(AppLog.T.POSTS, "photo picker > invalid position in getItemAtPosition");
-            return null;
-        }
-        return mMediaList.get(position);
-    }
-
-    private boolean isValidPosition(int position) {
```

Comment:
As far as I could tell this was only checking whether the framework (the adapter) was not touching any data outside of the data list. This should never happen if the adapter is clean (if it does, we wouldn't be able to rely on the framework to selecting the correct item because we wouldn't be able to rely on the position). I think it was there because the previous code got quite messy. This was mostly checking we're not touching anything in an empty list.